.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,77 @@
+---
+name: Bug
+about: For when something is there, but doesn't work how it should.
+
+---
+
+<!--- Please leave this line, it helps our automation: [issue-type:bug-report] --->
+<!--- Please keep this note for the community --->
+
+### Community Note
+
+* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
+* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
+* If you are interested in working on this issue or have submitted a pull request, please leave a comment
+* If the issue is assigned to the "modular-magician" user, it is either in the process of being autogenerated, or is planned to be autogenerated soon. If the issue is assigned to a user, that user is claiming responsibility for the issue. If the issue is assigned to "hashibot", a community member has claimed the issue already.
```

Comment:
What about using "If this issue"/"If an issue"? "If the issue" seems a little awkward to me especially when we refer to it as "it" later.